syntax = "proto2";

package hina_ssi_msgs.msgs;

import "vector3d.proto";
import "vector2d.proto";

message Soil
{
  required uint32     len_row = 1;
  required uint32     len_col = 2;
  repeated            gazebo.msgs.Vector3d chunk_field = 3;
  repeated            gazebo.msgs.Vector2d id_field = 4;
}
